modules/ocf_irc/manifests/ircd.pp

```diff
@@ -8,10 +8,10 @@
   }
 
   ocf::systemd::override { 'start-inspircd-before-anope':
-    service => 'inspircd',
-    content => "[Unit]\nBefore=anope.service",
+    unit    => 'inspircd.service',
+    content => "[Unit]\nBefore=anope.service\n",
     require => Package['inspircd', 'anope'],
-    before  => Service['inspircd'],
+    notify  => Service['inspircd'],
```

Comment:
is there a guarantee that this notify happens after systemd gets reloaded? that's the reason `before` was used instead of notify.